opentracing/ext/tags.py

```diff
@@ -25,17 +25,27 @@
 # PeerXXX tags can be emitted by either client-side of server-side to describe
 # the other side/service in a peer-to-peer communications, like an RPC call.
 
-# PeerService records the service name of the peer
+# PEER_SERVICE records the service name of the peer
 PEER_SERVICE = 'peer.service'
 
-# PeerHostname records the host name of the peer
+# PEER_HOSTNAME records the host name of the peer
 PEER_HOSTNAME = 'peer.hostname'
 
-# PeerHostIPv4 records IP v4 host address of the peer
+# PEER_HOST_IPV4 records IP v4 host address of the peer
 PEER_HOST_IPV4 = 'peer.ipv4'
 
-# PeerHostIPv6 records IP v6 host address of the peer
+# PEER_HOST_IPV6 records IP v6 host address of the peer
 PEER_HOST_IPV6 = 'peer.ipv6'
 
-# PeerPort records port number of the peer
+# PEER_PORT records port number of the peer
 PEER_PORT = 'peer.port'
+
+# SPAN_KIND hints at relationship between spans, like parent/child
+SPAN_KIND = 'span.kind'
+
+# Marks a span representing the client-side of an RPC or other remote call
+SPAN_KIND_RPC_CLIENT = 1
```

Comment:
ah, fair enough, I was originally going to use auto-enumeration with `range(3)`, but changed to exact constants to be able to write documentation. I'll change to letters.

FWIW, in my tracer implementation these tags will not actually be sent over the wire, instead the tracer intercepts them and uses to decide between ss/sr and cs/cr annotations. The same happens with all of the PEER tags above, they are used to populate Zipkin's internal `Endpoint` structure. We may want to put this into documentation somewhere.